staking/availability/measure.go

```diff
@@ -220,10 +239,10 @@ func SetInactiveUnavailableValidators(
 			continue
 		}
 
-		if d := new(big.Int).Sub(now, snapEpoch); d.Cmp(common.Big1) != 0 {
+		if d := new(big.Int).Sub(nowEpoch, snapEpoch); d.Cmp(common.Big1) != 0 {
```

Comment:
again, new validator could have snapEpoch==0. What is the reason to do this check?

Reply:
Because it’s all elected validators but that isn’t strictly the ones in committee selection